client/jetpack-connect/plans.jsx

```diff
@@ -196,8 +196,6 @@ class Plans extends Component {
 	}
 
 	selectFreeJetpackPlan() {
-		// clears whatever we had stored in local cache
-		this.props.selectPlanInAdvance( null, this.props.selectedSiteSlug );
```

Comment:
Why don't we `clearPlan()`?

Reply:
I wasn't quite sure why this was happening here. Now I see that this path doesn't go through `completeFlow()`.

Reply:
Reinstated.